source/Quantities.Units/Quantities.Units.csproj

```diff
@@ -10,7 +10,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Atmoos.Quantities" Version="1.1.1" />
+    <ProjectReference Include="..\Quantities\Quantities.csproj" />
+    <!--PackageReference Include="Atmoos.Quantities" Version="1.1.1" /-->
```

Comment:
why keep this comment?

Reply:
This is there to remind me to swith back to referencing the nuget. I'll need to split this PR into two. One for stuff only in `Quantities` for the release of `Atmoos.Quantities V 1.2`  only. The other PR for all other libraries that depend on `Atmoos.Quantities`.

This is one of the reason this is only a draft PR...

Reply:
I should have added a comment ;-)